docs/src/main/sphinx/connector/jdbc-common-configurations.fragment

```diff
@@ -25,3 +25,9 @@ connector:
     - Cache the fact that metadata, including table and column statistics, is
       not available
     - False
+  * - ``write.batch-size``
+    - Maximum number of rows to write in a single write operation batch.
+      Do not change this setting from default outside of test environments.
```

Comment:
Maybe just

Do not change this setting from the default. Updates may produce impact performance negatively.


I am still not 100% but this would be better imho

Reply:
That sounds ok by me too, I think it depends on how heavily we want to discourage users from changing this value, and how much technical detail we want to give as to why. I at least think it's important to call out the performance (and/or network) impact specifically, as that'll help people who're troubleshooting and looking for their symptoms.

I'll throw that up to @electrum and @hashhar  for their thoughts.

Reply:
I think let's borrow the wording from Hibernate docs which I find very concise and clear:

```
Maximum number of statements that will be batched together before executing the batch
```

I also like Manfred's suggested wording - `Do not change this setting from the default. Non-default values may impact performance negatively.`

Reply:
"Maximum number of statements in a batched execution."  Can't use should/will/etc